weblate_web/static/style-rtl.css

```
@@ -319,7 +319,9 @@ a.button:hover, input.button:hover {
 }
 a.button.inline, input.button.inline {
 	display: inline-block;
-	margin: 0 15px;
+	margin-bottom: 10px;
```

What is reason to decreasing left and right margins? Also I see no reason for splitting up it to individual tags.

```suggestion
	margin: 0 15px 10px;
```